src/components/PostPageVolunteerHeader/index.css

```diff
@@ -0,0 +1,18 @@
+.headerPicture {
+  width: 100%;
+  height: auto;
+}
+
+.postHeaderText p {
+  display: flex;
+  align-items: center;
+  font-family: "Acme";
```

Comment:
Ola added this to the App.css so it is implemented for all the code so there is not need to mention it in each css file

Reply:
When I removed it, it made the font change therefore I kept it but deleted the import